src/main/java/spireMapOverhaul/zones/divinitiesgaze/divinities/impl/Torch.java

```diff
@@ -0,0 +1,51 @@
+package spireMapOverhaul.zones.divinitiesgaze.divinities.impl;
+
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.status.Burn;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.vfx.UpgradeShineEffect;
+import com.megacrit.cardcrawl.vfx.cardManip.ShowCardBrieflyEffect;
+import spireMapOverhaul.zones.divinitiesgaze.cards.GuidingLight;
+import spireMapOverhaul.zones.divinitiesgaze.divinities.BaseDivineBeing;
+
+import java.util.List;
+import java.util.function.Consumer;
+import java.util.function.Supplier;
+import java.util.stream.Collectors;
+
+public class Torch extends BaseDivineBeing {
+
+  public Torch() {
+    super(GuidingLight.ID, Burn.ID);
+  }
+
+  @Override
+  public Consumer<Integer> doEventButtonAction() {
+    return (x) -> {
+      AbstractDungeon.player.heal((int)Math.floor(AbstractDungeon.player.maxHealth * 0.10f));
```

Comment:
Not sure if you really want to do anything but events typically become weaker at a certain asc level